clients/python/lakefs_client/client.py

```diff
@@ -27,18 +29,15 @@ def __init__(self, configuration=None, header_name=None, header_value=None, cook
             configuration = LakeFSClient._ensure_endpoint(configuration)
         self._api = _WrappedApiClient(configuration=configuration, header_name=header_name,
                                           header_value=header_value, cookie=cookie, pool_threads=pool_threads)
-        self.actions = ActionsApi(self._api)
-        self.auth = AuthApi(self._api)
-        self.branches = BranchesApi(self._api)
-        self.commits = CommitsApi(self._api)
-        self.config = ConfigApi(self._api)
-        self.health = HealthCheckApi(self._api)
-        self.metadata = MetadataApi(self._api)
-        self.objects = ObjectsApi(self._api)
-        self.refs = RefsApi(self._api)
-        self.repositories = RepositoriesApi(self._api)
-        self.staging = StagingApi(self._api)
-        self.tags = TagsApi(self._api)
+        for key, value in inspect.getmembers(sys.modules['lakefs_client.apis'], inspect.isclass):
+            name = key.lower()
+            if not name.endswith('api'):
+                continue
+            api_instance = value(self._api)
+            attr_name = name[0:len(name)-3]
```

Comment:
This is somewhat scary.  Can we instead use substitution?  Something like
```suggestion
            attr_name = re.sub('api$', '', name)
            if attr_name == name:
                continue
            api_instance = value(self._api)
```
might be better, as it makes it says "api" only once and avoids the magic number `3`.

Reply:
Simplified the code and used constant - I think it is better than having sub regex.
Only in Python 3.9 `removesuffix` was added :/